### PR TITLE
Sl stuff

### PR DESCRIPTION
**Pull request by atareshawty**
Absolutely do not pollute master with this garbage

### CODE REVIEW

File: controllers/Challengers.js [hunk 1]
```diff
@@ -49,6 +49,61 @@ function ChallengersController() {
     });
   };
 
+  this.showChallengeEvalForm = (req, res) => {
+    const mock = {
```
Comment (atareshawty) [inline controllers/Challengers.js]: lololololololololololololololololololol

File: controllers/Challengers.js [hunk 2]
```diff
@@ -49,6 +49,61 @@ function ChallengersController() {
...
+          person2: 'Brian Drake',
+          SpotId: 'L8'
+        }
+      ]
+    };
+    res.render('challengeEvalForm', mock);
+  };
+
+  this.challengeEval = (req, res) => {
+    console.log(req.body);
```
Comment (atareshawty) [inline controllers/Challengers.js]: For experimental purposes

File: controllers/Challengers.js [hunk 3]
```diff
@@ -49,6 +49,61 @@ function ChallengersController() {
...
+    const mock = {
```
Comment (atareshawty) [inline controllers/Challengers.js]: 😹

File: views/challengeEvalForm.handlebars [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<h2>Challenges for: {{PerformanceName}}</h2>
+{{#each challenges}}
+  <hr />
+  <div class="container">
+    <form class="form-inline" method="POST" action="/challengeeval">
```
Comment (atareshawty) [inline views/challengeEvalForm.handlebars]: forms are hard

**Comment by atareshawty** [comment]
# All of this is hard coded! Please don't let this last much longer after the squad leader retreat

**Comment by tarebyte** [comment]
:rocket: merge

**Pull request merged**